### PR TITLE
Feature/create documentation

### PR DESCRIPTION
**Pull request by coppetti**
I've started our documentation. Please review :)

### CODE REVIEW

**Review by m-maga — APPROVED** [review]

File: docs/ways-of-working.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+3. code was approved by Product Owner and lead member of technical staff related to the task
+
+### Tips for the Reviewer
+
+* critique code instead of people â€“ be kind to the coder, not to the code.
+* treat people who know less than you with respect, deference, and patience.
+* the only true authority stems from knowledge, not from position.
+* please note that Review Meetings are NOT problem solving meetings.
+* ask questions to clarify the reasoning behind the code instead of making accusatory statements.
+* avoid the â€œWhyâ€ questions.
```
Comment (h314to) [inline docs/ways-of-working.md]: Strange special characters here.

File: docs/ways-of-working.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+* remember to praise.
+* make sure you have good coding standards to reference.
+* remember that there is often more than one way to approach a solution.
+* you shouldnâ€™t rush through a code review - but also, you need to do it promptly.
```
Comment (h314to) [inline docs/ways-of-working.md]: Strange special characters here.

File: docs/ways-of-working.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+```
+
+This proccess will:
+
+* update the topic branch with any possible change on master
+* force the topic branch push to the origin
+
+#### 5. Open a Pull Request
+
+Before merging a topic branch into the mainline a pull request must be open. See [Principles for Evaluating Pull Requests](docs/pull-request-evaluation.md).
```
Comment (h314to) [inline docs/ways-of-working.md:194]: fix link

File: README.md [hunk 4]
```diff
@@ -1 +1,16 @@
-# documentation
\ No newline at end of file
+# README
+## Project Description
```
Comment (alesr) [inline README.md:2]: I would use just `# Documentation` and remove ` # README`  and  ` # Project Description`

**Review by JVieira13 — APPROVED** [review]
\m/

File: docs/project-members.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+Sara Magalhães - Product Owner
+
+Alessandro Resta - Scrum Master
+
+Filipe Agapito 
+
+Jorge Vieira
+
+Matheus Silveira
```
Comment (alesr) [inline docs/project-members.md:11]: Architect ...

File: docs/ways-of-working.md [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+```
+
+
+## Gitflow
+
+#### 1. Create topic branch
+
+```
+git fetch
+git checkout <branch-created-on-jira>
```
Comment (h314to) [inline docs/ways-of-working.md]: Not in Jira

**Review by h314to — APPROVED** [review]
Some minor typos, but it good to merge after they are fixed.

File: docs/ways-of-working.md [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+* [Code review](#markdown-header-code-review)
+* [Release](#markdown-header-release)
+* [Gitflow](#markdown-header-gitflow)
+
+
+
+## Overview
+
+This project is version tracked with Git and available in the [Spike Force 1 - Bacon Evaluators](https://github.com/spike-force-1-bacon-evaluators) organisation on GitHub.
+
```
Comment (alesr) [inline docs/ways-of-working.md:18]: Add reference/link to Trello board

File: docs/ways-of-working.md (hunk 3 above)
Comment (coppetti) [reply docs/ways-of-working.md:194]: fix why?

Comment (h314to) [reply docs/ways-of-working.md:194]: Nevermind. It's fine. My mistake.

**Pull request merged**